walkthroughs/1A-integrate-event-and-api-driven-apps/walkthrough.adoc

```diff
@@ -435,6 +435,7 @@ After setting up the integration between the Order Entry System and Order Manage
 :spring-url: http://spring-boot-rest-http-crud-spring-app.apps.city.openshiftworkshop.com/
 :fuse-url: https://eval.apps.city.openshiftworkshop.com/
 
+// TODO: currently the app has the login screen, either remove the screen in the app and allow free access, or instruct the user here to enter a username ([A-Za-z]{1,15}) and click *Log in*.
```

Comment:
Yeah, I meant to remove that. Will remove it before the lab session, thanks for the reminder!